packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -835,6 +915,10 @@ contract Deploy is Deployer {
             gameTypeString = "OP Succinct";
         } else if (rawGameType == GameTypes.KAILUA.raw()) {
             gameTypeString = "Kailua";
+        } else if (rawGameType == GameTypes.SUPER_CANNON.raw()) {
+            gameTypeString = "Super Cannon";
+        } else if (rawGameType == GameTypes.PERMISSIONED_SUPER_CANNON.raw()) {
```

Comment:
same my comment above. I don't think that the `PERMISSIONED_SUPER_CANNON` should be deployable.